### DISCUSSION TITLE
Question about scalability

### DISCUSSION THREAD

**Discussion by zim32** [vikejs/vike#581] [Discussion] [answered]
I have a question about serialised data and prerendering. Our site is very huge, it contains thousands of pages.

I have two questions:

1) is it possible to rerender in SSG only some pages?
2) is page data which is serialised between server and client loaded only per page, or this data is somehow merged for all pages and send only once?

**Comment by brillout** [comment, marked as answer]
https://vite-plugin-ssr.com/prerender-config#partial
https://vite-plugin-ssr.com/pageContext

**Comment by zim32** [comment]
I setup and test. All is working as I expected. Thanks for great project.
